documentation/asciidoc/computers/configuration/configuring-networking.adoc

```diff
@@ -1,183 +1,179 @@
 == Configuring Networking
 
-A GUI is provided for setting up wireless connections in Raspberry Pi OS with desktop. However if you are using Raspberry Pi OS Lite, you can set up wireless networking from the command line.
+Raspberry Pi OS provides a Graphical User Interface (GUI) for setting up wireless connections. Users of Raspberry Pi OS Lite and headless machines can set up wireless networking from the command line with https://developer-old.gnome.org/NetworkManager/stable/nmcli.html[`nmcli`].
+
+NOTE: These instructions work for Raspberry Pi OS _Bookworm_ and later. For earlier versions, check an older version of the documentation.
```

Comment:
IIRC we generally _only_ document the latest version of Raspberry Pi OS? (and we don't really provide any mechanisms for users to "check an older version").